lib/capnp_c.h

```diff
@@ -22,6 +28,19 @@
 typedef intmax_t ssize_t;
 #endif
 
+// Macro UNUSED(x) silences compiler warnings about unused parameter or variable `x`.
+#ifndef UNUSED
+#define UNUSED(x) (void)(x)
+#endif
+
+// Cross-platform macro ALIGNED_(x) aligns a struct by `x` bits.
```

Comment:
OK, great. I'll push a commit with 8 and also update the comment in `capn-malloc.c`.

It's early days on the microcontroller project so I'm not sure how much RAM I have to spare. The chip I have has 512KB RAM on board, so hopefully capnproto won't be a problem.